apps/marsha/templates/app/dc.yml.j2

```diff
@@ -38,8 +38,10 @@ spec:
               value: "{{ marsha_postgresql_port }}"
             - name: DJANGO_ALLOWED_HOSTS
               value: "{{ marsha_host | blue_green_hosts }}"
+{% if DJANGO_CLOUDFRONT_PRIVATE_KEY_PATH is defined %}
```

Comment:
I think `DJANGO_CLOUDFRONT_PRIVATE_KEY_PATH` is always defined (with a good default). Does it matter if it is defined while `DJANGO_CLOUDFRONT_PRIVATE_KEY` is not? If not, this block condition should be removed.

Reply:
:+1: